temfpy/uncertainty_quantification.py

```diff
@@ -1,38 +1,122 @@
-    evaluation makes it a commonly used function for testing a wide variety of methods in
-    computer experiments.
-    """
-    assert len(x) == 8
-
-    r_w = x[0]
-    r = x[1]
-    T_u = x[2]
-    H_u = x[3]
-    T_l = x[4]
-    H_l = x[5]
-    L = x[6]
-    K_w = x[7]
-
-    a = 2 * math.pi * T_u * (H_u - H_l)
-    b = np.log(r / r_w)
-    c = (2 * L * T_u) / (b * r_w ** 2 * K_w)
-    d = T_u / T_l
-
-    rslt = a / (b * (1 + c + d))
-    return rslt
-
-
-def ishigami(x, a=7, b=0.1):
-    """
-    The Ishigami function of Ishigami & Homma (1990) is used as an example for uncertainty and
-    sensitivity analysis methods, because it exhibits strong nonlinearity and nonmonotonicity.
-    """
-    assert len(x) == 3
-
-    rslt = (1 + b * x[2] ** 4) * np.sin(x[0]) + a * np.sin(x[1]) ** 2
-    return rslt
+"""Test capabilities for uncertainty quantification.
+
+This module contains a host of test models and functions often used in the uncertainty
+quantification literate.
+
+"""
+import numpy as np
+import math
+
+
+def borehole(x):
+    """The Borehole function models water flow through a borehole. Its simplicity and quick
+    evaluation makes it a commonly used function for testing a wide variety of methods in
+    computer experiments.
+    """
+    assert len(x) == 8
+
+    r_w = x[0]
+    r = x[1]
+    T_u = x[2]
+    H_u = x[3]
+    T_l = x[4]
+    H_l = x[5]
+    L = x[6]
+    K_w = x[7]
+
+    a = 2 * math.pi * T_u * (H_u - H_l)
+    b = np.log(r / r_w)
+    c = (2 * L * T_u) / (b * r_w ** 2 * K_w)
+    d = T_u / T_l
+
+    rslt = a / (b * (1 + c + d))
+    return rslt
+
+
+def ishigami(x, a=7, b=0.1):
+    """
+    The Ishigami function of Ishigami & Homma (1990) is used as an example for uncertainty and
+    sensitivity analysis methods, because it exhibits strong nonlinearity and nonmonotonicity.
+    """
+    assert len(x) == 3
+
+    rslt = (1 + b * x[2] ** 4) * np.sin(x[0]) + a * np.sin(x[1]) ** 2
+    return rslt
+
+
+def eoq_harris(x, r=10):
+    r"""Economic order quantity model.
+
+    This function computes the optimal economic order quantity (EOQ) based on the model presented in
+    [1]_. The EOQ minimizes the holding costs as well as ordering costs. The core parameters of
+    the model are the units per months `x[0]`, the unit price of items in stock `x[1]`,
+    and the setup costs of an order `x[2]`. The annual interest rate `r` is treated as an
+    additional parameter.
+
+    Parameters
+    ----------
+    x : array_like
+        Core parameters of the model
+
+    r : float, optional
+        Annual interest rate
+
+    Returns
+    -------
+
+    float
+        Optimal order quantity
+
+    Notes
+    -----
+
+    A historical perspective on the model is provided by [2]_. A brief description with the core
+    equations is available in [3]_.
+
+    References
+    ----------
+
+    .. [1] Harris, F. W. (1990).
+        How Many Parts to Make at Once.
```

Comment:
We are not consistent here with the capitalization of the titles. Here we have all capitalized but in [3] it is not. Please follow instructions here: https://github.com/OpenSourceEconomics/bibliography  If that does not have all the information you need than that is our fault and please work with @luward  to expand the description accordingly.

Reply:
If I raise an issue, please do not mark as resolved yourself, just comment that you addressed it and then I can look at it an mark as resolved. Why is now all capitalized? This is not in line with description in our bib repository? Let me know if I am missing sth. If we need to change it here, please also adjust in the resources section. Regarding the superscript issue, great, thanks!